package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "polymer-tools",
+  "private": true,
+  "description": "Polymer tooling monorepo!",
+  "homepage": "https://github.com/Polymer/tools",
+  "bugs": "https://github.com/Polymer/tools/issues",
+  "license": "BSD-3-Clause",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Polymer/tools.git"
+  },
+  "scripts": {
+    "bootstrap": "ls packages | sed \"s/^/packages\\\\//\" | sed \"s/$/\\\\/node_modules/\" | xargs rm -rf && lerna bootstrap",
```

Comment:
sed and xargs aren't available on Windows normally are they? Do we assume our windows users have some extra GNU tools installed? I know we often make unixy assumptions in our scripts so I guess no big deal for this PR.

Reply:
We generally assume unixy tools are installed. This is tested on appveyor, which is I think is a good starting point.

I'd like to have a clearer description of what exactly we require, but the appveyor tests are definitely running and working, as they already found one issue!